### PR TITLE
ReadMe update

### PR DESCRIPTION
**Pull request by SteveLLamb**
update readme to include dates

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
 
 ## Project details
 
 This document specifies constraints on SMPTE ST 377-1 for use in D-Cinema applications. It is intended to replace references to SMPTE ST 377:2004, allowing D-Cinema to benefit from the improvements of ST 377-1:2019 while retaining compatibility with ST 377:2004
 
 [GitHub issues](https://github.com/SMPTE/st429-20/issues) are preferred for discussion of this specification. Alternatively, comments can be sent to [21dc-chair@smpte.org](mailto:21dc-chair@smpte.org).
-```
 
+```
 See AG-22, Annex D for additional requirements specific to Public Committee Drafts.
```
Comment (palemieux) [inline README.md:15]: > See AG-22, Annex D for additional requirements specific to Public Committee Drafts.

This notice should be removed.

File: README.md [hunk 2]
```diff
@@ -1,13 +1,15 @@
 # ST 429-20 1ED (1st edition)
 Public CD of [SMPTE ST 429-20 1ED](21DC-CD-ST-429-20-202X-1ED.pdf)
 
+This document is made available for a public review period ending no earlier than `2022-02-01`, and no later than `2022-08-01`.
```
Comment (palemieux) [inline README.md:4]:
```suggestion
This document is made available for a public review period ending no earlier than `2022-01-23`, and no later than `2022-07-23`.
```

The PCD was published on 2021-07-23 if I am not mistaken.

**Review by palemieux — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (SteveLLamb) [reply README.md:15]: addressed in 2ecbad7

File: README.md [hunk 3]
```diff
@@ -1,13 +1,15 @@
...
 
 ## Project details
 
 This document specifies constraints on SMPTE ST 377-1 for use in D-Cinema applications. It is intended to replace references to SMPTE ST 377:2004, allowing D-Cinema to benefit from the improvements of ST 377-1:2019 while retaining compatibility with ST 377:2004
 
 [GitHub issues](https://github.com/SMPTE/st429-20/issues) are preferred for discussion of this specification. Alternatively, comments can be sent to [21dc-chair@smpte.org](mailto:21dc-chair@smpte.org).
-```
 
+```
 See AG-22, Annex D for additional requirements specific to Public Committee Drafts.
```
Comment (palemieux) [inline README.md:15]:
```suggestion
```

Comment (SteveLLamb) [reply README.md]: updated in e1a0ac4, sorry there was a GH caching issue apparently.

**Review by palemieux — APPROVED** [review]

**Pull request merged**